.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Issues covered
+Add reference(s) to a related issue in your repository.
+
+## Description
+A description of the changes proposed in the pull request. Explain what it does to give the reviewer some context for what they're reviewing.
+
+## How to test
+1. Navigate to...
+2. Tap...
+Any testing instruction
+
+## Screenshots/Video
+Post screenshots or video showing your changes, ideally showing how the app worked before and after these changes. Delete this section if this PR contains no visual changes.
```

Comment:
I don't find this section useful as a reviewer, because I'm going to build and run the app to see for myself. It's also time consuming to create. Are others finding value in this? @martindsq @dcadenas @joshuatbrown

Reply:
I do use it as a reviewer. I always review the code first, before building and running. When there are screenshots or video, I have a better idea of what I'm reviewing.

As a PR author, I don't find it time consuming to create. I generally test manually a bunch of times; during one of those, I hit the Save Screen or Record Screen option in the Simulator and upload it to the PR.

Reply:
I think this is a good idea as well